sbws/lib/resultdump.py

```diff
@@ -455,6 +495,9 @@ def store_result(self, result):
                 self.data[fp] = []
             self.data[fp].append(result)
             self.data = trim_results(self.fresh_days, self.data)
+            # we probably do not want to remove the results for a relay
+            # that has changed address when storing the results
+            # it will be already done when loading
```

Comment:
Right.

Please add to this comment the fact that we are not calling `trim_results_ip_changed` because [reasons you already stated]